pennylane/ops/functions/simplify.py

```diff
@@ -15,22 +15,27 @@
 This module contains the qml.simplify function.
 """
 from copy import copy
+from functools import wraps
+from typing import Callable, Union
 
 import pennylane as qml
+from pennylane.measurements import MeasurementProcess
 from pennylane.operation import Operator
+from pennylane.qnode import QNode
+from pennylane.queuing import QueuingContext
+from pennylane.tape import QuantumTape, stop_recording
 
 
-@qml.op_transform
```

Comment:
This decorator cannot be used because of [this reason](https://github.com/PennyLaneAI/pennylane/pull/2854#issuecomment-1203933458).